cpp/src/arrow/CMakeLists.txt

```diff
@@ -514,8 +516,13 @@ if(ARROW_IPC)
   add_arrow_test(extension_type_test)
 endif()
 
-add_arrow_test(memory_pool_test)
-add_arrow_test(pretty_print_test)
+add_arrow_test(misc_test
```

Comment:
```suggestion
add_arrow_test(core_types_test
```
?

Reply:
"Core types" sounds like "datatypes" to me, so it looks a bit misleading.

Reply:
I agree "misc" isn't satisfactory :-)